lizrd/scripts/grid.py

```diff
@@ -63,7 +63,6 @@
     PROCESS_CALL_FUNCTION = lambda args, env: subprocess.run(
         [str(arg) for arg in args if arg is not None], env=env
     )
-    AUXILIARY_PROCESS_CALL_FUNCTION = None
```

Comment:
Why delete this?

Reply:
no reason to have it, artifact from previous implementation of babyslurm